corehq/apps/hqwebapp/static/hqwebapp/js/main.js

```diff
@@ -325,7 +325,7 @@ var COMMCAREHQ = (function () {
 function setUpIeWarning() {
     'use strict';
     var $warning;
-    if ($.browser.msie) {
+    if (/msie/i.test(navigator.userAgent) || /windows/i.test(navigator.userAgent)) {
```

Comment:
yes, I would say so. IE 10 and above are actually reasonable...

On Fri, Sep 11, 2015 at 3:23 PM, Jenny Schweers <notifications@github.com>
wrote:

> In corehq/apps/hqwebapp/static/hqwebapp/js/main.js
> <https://github.com/dimagi/commcare-hq/pull/7499#discussion_r39307483>:
>
> > @@ -325,7 +325,7 @@ var COMMCAREHQ = (function () {
> >  function setUpIeWarning() {
> >      'use strict';
> >      var $warning;
> > -    if ($.browser.msie) {
> > +    if (/msie/i.test(navigator.userAgent) || /windows/i.test(navigator.userAgent)) {
>
> Apparently IE comments aren't supported after IE9 - is it only older
> versions of IE that we're concerned with?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/commcare-hq/pull/7499/files#r39307483>.
>